### PR TITLE
Port dialog widget to react

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

File: src/app/pages/partners/partner-details/partner-details.js [hunk 1]
```diff
@@ -68,7 +65,7 @@ function Overview({model, icon}) {
     // }
     // console.info('Images', images);
     // TESTING
-
+    console.info('Model has url?', model);
     return (
```
Comment (nathanstitt) [inline src/app/pages/partners/partner-details/partner-details.js:69]: console.log left in 😁

File: src/app/pages/partners/partner-details/partner-details.scss [hunk 2]
```diff
@@ -24,10 +24,10 @@
     .scrolling-region {
         background-color: ui-color(white);
         height: 100%;
-        max-width: 100%;
+        max-width: $content-max;
```
Comment (nathanstitt) [inline src/app/pages/partners/partner-details/partner-details.scss:27]: I don't see $content-max var defined anywhere but it's probably a global

File: test/src/pages/institutional-partnership/participants.test.js [hunk 3]
```diff
@@ -3,16 +3,14 @@ import shellBus from '~/components/shell/shell-bus';
...
-        const clickables = wrapper.find('.show-established-partners');
 
-        expect(clickables).toHaveLength(1);
-        const dialogShown = new Promise((resolve) => {
-            shellBus.on('showDialog', resolve, 'once');
-            clickables.at(0).simulate('click');
-        });
-
-        return dialogShown;
+        done();
```
Comment (nathanstitt) [inline test/src/pages/institutional-partnership/participants.test.js:9]: I'm confused by why `done` was needed here, I don't see where makeMountRender
 is being waited on.  I also typically just make the it block `async` and wait vs using `done`

File: test/src/pages/institutional-partnership/participants.test.js [hunk 4]
```diff
@@ -3,16 +3,14 @@ import shellBus from '~/components/shell/shell-bus';
...
+        // const clickables = wrapper.find('.show-established-partners');
+        //
+        // expect(clickables).toHaveLength(1);
+        // shellBus.on('showDialog', done, 'once');
+        // clickables.at(0).simulate('click');
```
Comment (nathanstitt) [inline test/src/pages/institutional-partnership/participants.test.js:14]: commented out code

**Review by nathanstitt — APPROVED** [review]
looks good Roy,  just a few minor things

File: src/app/pages/partners/partner-details/partner-details.scss (hunk 2 above)
Comment (RoyEJohnson) [reply src/app/pages/partners/partner-details/partner-details.scss:27]: It's in the pattern library.

**Pull request merged**